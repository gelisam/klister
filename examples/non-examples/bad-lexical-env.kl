#lang kernel

(define f (lambda (ff) y))
(define g (lambda (y) (f)))

(example (g 2))
