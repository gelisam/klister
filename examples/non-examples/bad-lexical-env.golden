In Phase: p0
Unknown: #[bad-lexical-env.kl:3.24-3.25]<y>
